src/integrations/front-end-integration.php

```diff
@@ -471,7 +480,7 @@ private function maybe_remove_title_presenter( $presenters ) {
 		}
 
 		// Remove the title presenter if the theme is hardcoded to output a title tag so we don't have two title tags.
-		if ( ! \get_theme_support( 'title-tag' ) && ! $this->options->get( 'forcerewritetitle', false ) ) {
+		if ( $this->title_presenter_should_be_removed() ) {
```

Comment:
I think you might want to change this to be should_title_presenter_be_removed, to be more inline with the test of the codebase.

Reply:
Good idea! I'll change that some time today.